### PR TITLE
Admin tool to send priority announcements.

### PR DESCRIPTION
**Pull request by francinum**
set some vars make some noise or smth

## Changelog
:cl:
admin: New debug tool to send priority announcements.
/:cl:

### CODE REVIEW

**Comment by PowerfulBacon** [comment]
seems really annoying having to varedit it every single time, also cant you just use the centcom announce verb?

**Comment by francinum** [comment]
> seems really annoying having to varedit it every single time

Can you be more clear on what you mean by this?

> cant you just use the centcom announce verb?

For most cases the verb is just fine, for the case planned out here, no. It's best used when you have prep time and an empty thunderdome.

**Comment by jupyterkat** [comment]
needs an ui to set the vars to actually announce something, so mins don't have to open VV.

**Comment by francinum** [comment]
> needs an ui to set the vars to actually announce something, so mins don't have to open VV.

Honestly I wrote this as a quick and dirty tool for an event I completely missed, and I really, *really* do not want to be fucked writing UI code, so screw this.

**Comment by francinum** [comment]
actually nah, it doesn't fucking need one.

The window to set them *is* vv.

If that's a problem, please politely fix the sentience balloon first.

**Comment by utilbird** [comment]
at least make it use input()

**Comment by francinum** [comment]
> at least make it use input()

As I said before, There's no need for an even more annoying massive chain of input() calls when you can just set the vars by hand.

File: code/game/objects/items/event_tools.dm [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+
+//Allow them to be used from observer with a confirmation box.
+//Stolen with only some shame from fun_balloon code.
+/obj/item/patool/attack_ghost(mob/user)
+	if(!user.client || !user.client.holder)
+		return
+	var/confirmation = alert("Trigger Announcer [src]","Priority Announcer","Yes","No")
+	if(confirmation == "Yes")
+		priority_announce(a_text, a_title, a_sound, a_type, a_override, a_id)
```
Comment (Archanial) [inline code/game/objects/items/event_tools.dm]:
```suggestion
	if(alert("Trigger Announcer [src]","Priority Announcer","Yes","No") == "Yes")
		priority_announce(a_text, a_title, a_sound, a_type, a_override, a_id)
```

**Comment by ike709** [comment]
Seems annoying to use.

**Comment by francinum** [comment]
> Seems annoying to use.

Somewhat, It was made for a specific purpose and that is pre-writing or sending advanced priority announcements.

9 times out of 10 you would just use the `create-command-report` verb.

This is for those 1 times out of 10 where that is insufficient.

**Comment by utilbird** [comment]
I still don't see why you'd want to make it dependent on VV to use properly.

**Comment by francinum** [comment]
Because I don't really see a reason to bother with bashing together a chain of inputs when you can just use existing tooling. Regardless I'm redoing this anyways.

**Pull request closed without merging**